src/app/public/modules/flyout/flyout-iterator.component.html

```diff
@@ -1,25 +1,62 @@
 <div
   id="iterators"
+  class="sky-flyout-iterators"
 >
   <button
-    class="sky-btn sky-btn-default sky-margin-inline-compact"
+    class="sky-btn"
     [attr.aria-label]="'skyux_flyout_iterator_previous_button' | skyLibResources"
     [disabled]="previousButtonDisabled"
+    [ngClass]="{
+      'sky-btn-default':
+        (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'default',
+      'sky-btn-icon-borderless':
+        (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern',
+      'sky-margin-inline-compact':
+        (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'default',
+      'sky-margin-inline-sm':
+        (themeSvc?.settingsChange | async)?.currentSettings.theme.name === 'modern'
```

Comment:
Part of me thinks all four of these calls should be moved to the component class... Does `async` pipe cache results?

Reply:
Sorry, should have been more clear. I'm not concerned about it not working, it's mostly just a concern with the duplicate code in the templates. I know using `async` pipe is convenient, but since we're reading the same property multiple times in the template, wouldn't it be more efficient to set a public property (such as `themeName: string`) once, in the component class?
```
this.themeSvc.settingsChange.pipe(takeUntil(...)).subscribe(settings => {
  this.themeName = settings.currentSettings.theme.name;
});
```

Reply:
That makes more sense! Done.